python/beeai_framework/adapters/litellm/utils.py

```diff
@@ -56,6 +56,10 @@ def parse_extra_headers(
 
 def _parse_header_string(header_string: str) -> dict[str, str]:
     """Parses a comma-separated string of headers into a dictionary."""
+    from beeai_framework.logger import Logger
```

Comment:
I moved this inside the function because the new calls to `litellm_debug` in this file were causing a import loop for `logger`